src/lib/common/homestore_config.fbs

```diff
@@ -140,6 +140,9 @@ table Generic {
     // percentage of cache used to create indx mempool. It should be more than 100 to 
     // take into account some floating buffers in writeback cache.
     indx_mempool_percent : uint32 = 110;
+
+    // journal chunk pool size
+    journal_chunk_pool_size: uint32 = 5;
```

Comment:
can you please add unit as well, something like journal_chunk_pool_size_mb

Reply:
@hari, add num_

Reply:
Ack